spiceaidocs/docs/components/data-connectors/abfs.md

```diff
@@ -165,6 +179,10 @@ datasets:
     name: my_data
     params:
       abfs_tentant_id: B3E1A8F4-9D5B-4D3B-8D2E-1F4A9D5B4D3B
-      abfs_client_id: ${ secrets:MY_CLIENT_ID }
-      abfs_client_secret: ${ secrets:MY_CLIENT_SECRET }
-```
\ No newline at end of file
+      abfs_client_id: A587D13A-7E4E-46AB-BB87-E7A8AAFB42F3
+      abfs_client_secret: qoiwdjqidj213094103213o0~!!
+```
```

Comment:
Changes lgtm, just wonder why a real client_id & client_secret is used instead of ${ secrets:MY_CLIENT_ID } and ${ secrets:MY_CLIENT_SECRET } here

Reply:
Yeah, we show off best practices in our documentation to use the secrets support

Reply:
This is fixed, thanks for the suggestion